docs/docs/api/wafrules.md

```diff
@@ -0,0 +1,60 @@
+# WAF Rule
+
+A `kind: WebApplicationFirewallRule` resource will manage one or more WAF rules in a managed [zone](../zone).
+Each resource can contain a single WAF rule when specified under `spec.rule` vs `spec.rules`.
```

Comment:
why not just ask people to add a single entry in a list

Reply:
yeah, that does make more sense i think